mobile/lib/common/status_screen.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/bridge_generated/bridge_definitions.dart';
+import 'package:get_10101/common/service_status_notifier.dart';
+import 'package:provider/provider.dart';
+
+class StatusScreen extends StatefulWidget {
+  const StatusScreen({required this.fromRoute, super.key});
+
+  final String fromRoute;
+
+  @override
+  State<StatusScreen> createState() => _StatusScreenState();
+}
+
+class _StatusScreenState extends State<StatusScreen> {
+  @override
+  Widget build(BuildContext context) {
+    ServiceStatusNotifier serviceStatusNotifier = context.watch<ServiceStatusNotifier>();
+
+    final orderbookStatus =
+        statusToString(serviceStatusNotifier.getServiceStatus(Service.Orderbook));
+    final coordinatorStatus =
+        statusToString(serviceStatusNotifier.getServiceStatus(Service.Coordinator));
+    final overallStatus = statusToString(serviceStatusNotifier.overall());
+
+    return Scaffold(
+      appBar: AppBar(title: const Text("Status")),
+      body: SafeArea(
+        child: Column(
```

Comment:
🙃 might be worth adding some form of centered icon to the screen that shows the overall status in green, red or gray.

Reply:
I plan to do that in a follow-up!